pkgs/by-name/st/stalwart-mail/spam-filter.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenv,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```

Use the `finalAttrs` pattern instead (see https://github.com/NixOS/nixpkgs/pull/119942)

---

TIL, should I do those changes for all other stalwart-mail packages too then? I was just following the webadmin setup.